reference-implementation/web-platform-tests/readable-streams/general.js

```diff
@@ -85,7 +85,7 @@ test(() => {
     start(controller) {
       assert_equals(this, source, 'source is this during start');
 
-      const methods = ['close', 'enqueue', 'error', 'constructor'];
+      const methods = ['_cancel', '_pull', 'close', 'enqueue', 'error', 'constructor'];
```

Comment:
Done. I don't know if we can use a Symbol directly on a class syntax. Maybe not. So, the choices are to define the methods in the constructor or to place them out of the class block. I've chose the latter. WDYT?